src/Artemis.VisualScripting/Nodes/Text/StrıngNullOrEmpty.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Artemis.Core;
+
+namespace Artemis.VisualScripting.Nodes.Text;
+
+[Node("String Null or WhiteSpace", "Checks whether the string is null, empty or white space.", 
+    "Text", InputType = typeof(string), OutputType = typeof(bool))]
+public class StringNullOrWhiteSpaceNode : Node
```

Comment:
I'd rather have the display values of this node as follows
- Name: "Text is empty"
- Description: "Outputs whether the input text is empty"

Then it only has a single output pin that is just true if the string is null, is empty or only contains whitespaces
Basically just a single `string.IsNullOrWhiteSpace()` call.

The same philosophy applies here, I want to avoid the user having to worry about strings and whether they are null or empty of contain only whitespaces, text should just be text :)